xprof/xprof.sh.erb.in

```diff
@@ -448,11 +448,11 @@ setub_babeltrace_live() {
 <% if languages.include?("omp") %>
      --component=filter.ompinterval.interval \
 <% end %>
-     --component=sink.tally.tally \
-     --params="display=compact, \
```

Comment:
Also why changing for this one?
We may have more mode ? Like for MPI we may want to have only `min` / `max` rank